test/tests/functional/pbs_ralter.py

```diff
@@ -2263,3 +2263,36 @@ def test_resv_resc_assigned(self):
                                  confirm=False)
         self.server.expect(NODE, {'resources_assigned.ncpus': 4},
                            max_attempts=1, id=resv_node)
+
+    def test_alter_standing_resv_future_occrs(self):
+        """
+        Test that when start time of a confirmed standing reservation is
+        altered, only the upcoming occurence changes and not all occurences
+        are modified.
+        """
+
+        duration = 20
+        offset = 3600
+        shift = -3000
+
+        rid, start, end = self.submit_and_confirm_reservation(
+            offset, duration, select="2:ncpus=4", standing=True,
+            rrule="FREQ=HOURLY;COUNT=3")
+
+        self.server.expect(RESV,
+                           {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')},
+                           rid)
+        # move the reservation 10 mins in future
+        self.alter_a_reservation(rid, start, end, confirm=True, alter_s=True,
+                                 alter_e=True, shift=shift)
```

Comment:
I think the comment is incorrect, shift is -3600.  Won't this try and move the start time to now?  Actually -3600 will likely be in the past because some time has passed.  Don't you want something like -1800?

Reply:
No, shift is 3000. It's the offset that's set to 3600.